src/components/ContactDisplayScreen/ContactDisplayScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useEffect} from 'react';
+import {View, ScrollView} from 'react-native';
+import {useDispatch, useSelector} from 'react-redux';
+import {getContacts} from '../../actions/index';
+import Async from '../Async/Async';
+import ContactStyles from './ContactStyles';
+import ContactBox from './ContactBox';
+import {AppStore} from '../../types/index';
+
+const ContactDisplayScreen = () => {
+  const contacts = useSelector(
+    (state: AppStore) => state.contactsReducer.contacts,
+  );
+  const asyncState = useSelector(
+    (state: AppStore) => state.contactsReducer.asyncState,
+  );
+  const error = useSelector((state: AppStore) => state.contactsReducer.error);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getContacts());
+  }, [dispatch]);
```

Comment:
can leave this empty so it will be called only for `cDM`

Reply:
Removed